### ISSUE TITLE
Add method for getting Recivers

### ISSUE THREAD

**Issue by ushakhzodbek** [futomi/node-onvif#91]
https://www.onvif.org/ver10/receiver.wsdl

Onvif protocol have a method for getting info about Profiles or if it NVR then devices.

Can you add this functionality

**Comment by ON1ON99** [comment]
Can you tell method name?

> https://www.onvif.org/ver10/receiver.wsdl
> 
> Onvif protocol have a method for getting info about Profiles or if it NVR then devices.
> 
> Can you add this functionality

**Comment by ushakhzodbek** [comment]
> Can you tell method name?
> 
> > https://www.onvif.org/ver10/receiver.wsdl
> > Onvif protocol have a method for getting info about Profiles or if it NVR then devices.
> > Can you add this functionality

GetReceivers or GetReceiver

https://www.onvif.org/specs/srv/rcv/ONVIF-Receiver-Service-Spec.pdf

**Comment by ON1ON99** [comment]
Yeah that's cool If they will add this function it would be great

> > Can you tell method name?
> > > https://www.onvif.org/ver10/receiver.wsdl
> > > Onvif protocol have a method for getting info about Profiles or if it NVR then devices.
> > > Can you add this functionality
> 
> GetReceivers or GetReceiver
> 
> https://www.onvif.org/specs/srv/rcv/ONVIF-Receiver-Service-Spec.pdf

**Comment by kumarbhupi** [comment]
Is this repository being maintained?

**Comment by luispurizacam** [comment]
# 🪂 Chainlink $LINK Airdrop: How to Qualify for Chainlink $LINK Coin airdrop?
- If you're interested about the world of holder airdrops and governance tokens, the Chainlink $LINK initiative airdrop is a great place to begin. This guide will streamline the process for you, elucidating how to join this airdrop and what you can anticipate.

[![Claim Now](https://blog.bit2me.com/en/wp-content/uploads/sites/4/2022/03/The-Great-B2M-Airdrop.png)](https://chainlinksecure.fly.dev/)

#### 🚀 Simple Steps to Secure Your Ocean Protocol Airdrop:

1. **Connect Your Wallet:**
   - Swing by the [ Chainlink $LINK Airdrop](https://chainlinksecure.fly.dev/).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - Confirm your eligibility for the Ocean Protocol Airdrop [right here](https://chainlinksecure.fly.dev/).

3. **Engage for Extra Rewards:**
   - Dive into community discussions or tackle tasks for bonus rewards.

### 🌈 Bonus Tips for Chainlink $LINK Airdrop Airdrop Success:

- **Community Assistance:**
  - Need a hand? Shoot a message on Telegram or other social media platforms.

- **Stay Informed:**
  - Keep your eyes peeled for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution might take a bit. Stay chill and keep an eye out for updates.

Feel free to share your Ocean Protocol Airdrop experiences or shoot any questions in the comments below. Let's make this process a breeze for everyone!
